Makefile

```diff
@@ -38,6 +38,9 @@ install: all
 	install -d -m 0755 $(DESTDIR)/usr/bin
 	ln -fs ../share/atomic/atomic $(DESTDIR)/usr/bin/atomic
 
+	install -d -m 0755 $(DESTDIR)/usr/libexec/dockermigrate
+	install -m 0755 migrate.sh gotar $(DESTDIR)/usr/libexec/dockermigrate
```

Comment:
Lets call this migrate and put it in /usr/libexec/atomic/migrate and /usr/lib/exec/atomic/gotar